api/lib/spree/api/engine.rb

```diff
@@ -21,6 +22,12 @@ class Engine < Rails::Engine
         end
       end
 
+      initializer 'extend ActiveRecord with Spree::Webhooks' do |_app|
+        ActiveSupport.on_load(:active_record) do
+          Spree::Base.send :include, Spree::Webhooks::HasSpreeWebhooks
```

Comment:
@sebastian-palma this will also inject this code into `Spree::Webhooks` models as well - I think we need a more granular level of webhooks and inject this into specific models

Reply:
Might it be including the module directly on each model as needed?